src/theme/themes.ts

```diff
@@ -45,6 +45,7 @@ export const dark: SaleorThemeColors = {
     3: "rgba(250, 250, 250, 0.6)",
     4: "rgba(250, 250, 250, 0.4)",
     5: "rgba(250, 250, 250, 0.1)",
+    6: "rgba(250, 250, 250, 0.1)",
```

Comment:
5 and 6 are exactly the same, what's the point of that?

Reply:
This is because light theme has main[6] color but we don't have that for dark mode, so main.5 was reused instead ;)